### PR TITLE
Feature/407342 fowler welch

### PR DESCRIPTION
**Pull request by mehnpat1199**
# Pull Request Details

## What this PR does / why we need it:
*A brief description of changes being made*
*Link to the relevant work items: e.g: Relates to ADO Work Item AB#213700 and builds on #3376 (link to ADO Build ID URL)*

## Special notes for your reviewer
*Any specific actions or notes on review?*

## Testing
*Any relevant testing information and pipeline runs.*

## Checklist (please delete before completing or setting auto-complete)
- [ ] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [ ] Title pattern should be `{work item number}: {title}`
- [ ] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

## How does this PR make you feel:
gif

### CODE REVIEW

**Review by padget11 — APPROVED** [review]

File: test/unit/services/parser-service.test.js [hunk 1]
```diff
@@ -881,3 +881,199 @@ describe('parseTjmorris', () => {
...
+          D: 'Online Check',
+          E: 'Meursing Code',
+          F: 'Description of goods',
+          G: 'Country of Origin',
+          H: 'No. of pkgs',
+          I: 'Type of pkgs',
+          J: 'Total Gross Weight',
+          K: 'Total Net Weight',
+          L: 'Total Line Value',
+          M: 'NIIRMS Dispatch number',
```
Comment (jaybarden) [inline test/unit/services/parser-service.test.js:940]: Is this version of NIRMS supposed to have 2 "I"s? If so, please ignore me

**Review by jaybarden — COMMENTED** [review]
I added a comment in the test file but will add here to - there are a couple of places where NIRMS is NIIRMS - is that correct? Apologies if it is, it just looks wrong to me.

**Review by jaybarden — DISMISSED** [review]
All looks good to me and apologies again for the "NIRMS/NIIRMS" question

**Review by padget11 — APPROVED** [review]

File: app/services/parser-service.js [hunk 2]
```diff
@@ -349,6 +356,71 @@ function parseTjmorris (packingListJson) {
...
+      D: 'Online Check',
+      E: 'Meursing code',
+      F: 'Description of goods',
+      G: COUNTRY_OF_ORIGIN,
+      H: 'No. of pkgs ',
+      I: 'Type of pkgs',
+      J: 'Total Gross Weight',
+      K: 'Total Net Weight',
+      L: 'Total Line Value',
+      M: 'NIIRMS Dispatch number',
```
Comment (dynamicmonkeys) [inline app/services/parser-service.js:388]: Please change to NIRMS

File: test/unit/services/parser-service.test.js (hunk 1 above)
Comment (dynamicmonkeys) [inline test/unit/services/parser-service.test.js:940]: Please change to NIRMS

File: test/unit/services/parser-service.test.js [hunk 3]
```diff
@@ -881,3 +881,199 @@ describe('parseTjmorris', () => {
...
+          D: 'Online Check',
+          E: 'Meursing code',
+          F: 'Description of goods',
+          G: 'Country of Origin',
+          H: 'No. of pkgs \r\n(1547)',
+          I: 'Type of pkgs',
+          J: 'Total Gross Weight \r\n(11015.700kgs)',
+          K: 'Total Net Weight \r\n(7921.700kgs)',
+          L: 'Total Line Value \r\n(41662.4)',
+          M: 'NIIRMS Dispatch number',
```
Comment (dynamicmonkeys) [inline test/unit/services/parser-service.test.js:978]: Please change to NIRMS

File: test/unit/services/parser-service.test.js [hunk 4]
```diff
@@ -881,3 +881,199 @@ describe('parseTjmorris', () => {
...
+        D: 'Online Check',
+        E: 'Meursing code',
+        F: 'Description of goods',
+        G: 'Country of Origin',
+        H: 'No. of pkgs \r\n(1547)',
+        I: 'Type of pkgs',
+        J: 'Total Gross Weight \r\n(11015.700kgs)',
+        K: 'Total Net Weight \r\n(7921.700kgs)',
+        L: 'Total Line Value \r\n(41662.4)',
+        M: 'NIIRMS Dispatch number',
```
Comment (dynamicmonkeys) [inline test/unit/services/parser-service.test.js:1017]: Please change to NIRMS

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "trade-exportscore-plp",
-  "version": "0.1.25",
+  "version": "0.1.26",
```
Comment (dynamicmonkeys) [inline package.json:3]: Please merge in main and bump this

**Review by dynamicmonkeys — CHANGES_REQUESTED** [review]

File: app/services/parser-service.js (hunk 2 above)
Comment (dynamicmonkeys) [reply app/services/parser-service.js:388]: Ignore me, thanks @padget11 for info :)

File: test/unit/services/parser-service.test.js (hunk 1 above)
Comment (dynamicmonkeys) [reply test/unit/services/parser-service.test.js:940]: Ignore me, thanks @padget11 for info :)

File: test/unit/services/parser-service.test.js (hunk 3 above)
Comment (dynamicmonkeys) [reply test/unit/services/parser-service.test.js:978]: Ignore me, thanks @padget11 for info :)

File: test/unit/services/parser-service.test.js (hunk 4 above)
Comment (dynamicmonkeys) [reply test/unit/services/parser-service.test.js:1017]: Ignore me, thanks @padget11 for info :)

**Review by dynamicmonkeys — APPROVED** [review]

**Pull request merged**